graphql/bookings.graphql

```diff
@@ -1,36 +1,36 @@
 extend type Query {
-    bookings: [Booking!]! @paginate
-    bookingsByDate(from: Date! @where(clause: "whereDate")): [Booking!]! @all
-    bookingsBetweenDates(from: DateRange @whereBetweenDate): [Booking!]! @all
-    booking(id: ID @eq): Booking @find
+	bookings: [Booking!]! @paginate
+	bookingsByDate(from: Date! @where(clause: "whereDate")): [Booking!]! @all
+	bookingsBetweenDates(from: DateRange @whereBetweenDate): [Booking!]! @all
+	booking(id: ID @eq): Booking @find
 }
 
 extend type Mutation {
-    createBooking(
-        user_id: ID! @rules(apply: ["required", "App\\Rules\\UserExistsRule"]),
-        position: PositionRelation!,
-        from: DateTime! @rules(apply: ["before:to"]),
-        to: DateTime! @rules(apply: ["after:from"]),
-    ): Booking! @create
+	createBooking(
+		user_id: ID! @rules(apply: ["required", "App\\Rules\\UserExistsRule"]),
+		position_id: Int!,
```

Comment:
Why not keep this as `position: PositionRelation!,`? Seeing as `updateBooking` uses the relationship would be good to be consistent

Reply:
I felt the query was too verbose - why not just stick `position_id` in rather than having to maintain a type because you'd still have to check that the position exists, right?

Reply:
Fair enough - I think the type with the `connect` attribute automatically check it exists though? I don't really mind either way - realistically the position isn't going to change on update (apart from maybe for a mentoring session)

Reply:
I think we won't really know the best way untill we start consuming the API in the front-end